ansible/roles/products/common/resources/templates/scripts/setup-hand.j2

```diff
@@ -23,7 +23,7 @@ if [ -f /tmp/id_rsa ]; then
 fi
 if {{ launch_hand | lower }}; then
     if {{ sim_hand | lower }}; then
-        roslaunch sr_robot_launch srhand.launch
+        roslaunch sr_robot_launch srhand.launch sim:=true
```

Comment:
Why sim true

Reply:
Because this is in the conditional if sim_hand = true then... but it was always missing sim:=true so I fixed it here